opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/GizOpdProfileActivity.java

```diff
@@ -49,11 +72,38 @@ public boolean onOptionsItemSelected(MenuItem item) {
             showOpdTransferDialog(GizConstants.EventType.OPD_PNC_TRANSFER);
         } else if (id == R.id.opd_menu_item_enrol_maternity) {
             showOpdTransferDialog(GizConstants.EventType.OPD_MATERNITY_TRANSFER);
+        } else if (id == R.id.opd_menu_item_sync) {
+            startSync();
         }
 
         return true;
     }
 
+    protected void showStartSyncToast() {
+        Toast.makeText(this, getResources().getText(R.string.action_start_sync),
+                Toast.LENGTH_SHORT).show();
+    }
+
+    protected void startSync() {
+        if (!SyncStatusBroadcastReceiver.getInstance().isSyncing()) {
+            initiateSync();
+            showStartSyncToast();
+
+        } else
+            Toast.makeText(this, getResources().getText(R.string.sync_in_progress),
+                    Toast.LENGTH_SHORT).show();
+    }
+
+    protected void initiateSync() {
+        ImageUploadServiceJob.scheduleJobImmediately(ImageUploadServiceJob.TAG);
+        SyncServiceJob.scheduleJobImmediately(SyncServiceJob.TAG);
+        SyncSettingsServiceJob.scheduleJobImmediately(SyncSettingsServiceJob.TAG);
+        ZScoreRefreshIntentServiceJob.scheduleJobImmediately(ZScoreRefreshIntentServiceJob.TAG);
+        WeightIntentServiceJob.scheduleJobImmediately(WeightIntentServiceJob.TAG);
+        HeightIntentServiceJob.scheduleJobImmediately(HeightIntentServiceJob.TAG);
+        VaccineServiceJob.scheduleJobImmediately(VaccineServiceJob.TAG);
+    }
```

Comment:
Can we have a util for this?

Reply:
Do you need these for opd sync?
`        ZScoreRefreshIntentServiceJob.scheduleJobImmediately(ZScoreRefreshIntentServiceJob.TAG);
        WeightIntentServiceJob.scheduleJobImmediately(WeightIntentServiceJob.TAG);
        HeightIntentServiceJob.scheduleJobImmediately(HeightIntentServiceJob.TAG);
        VaccineServiceJob.scheduleJobImmediately(VaccineServiceJob.TAG);`

Reply:
Have added a util for this